distribution/lib/Standard/Base/0.0.0-dev/src/Data/Index_Sub_Range.enso

```diff
@@ -117,7 +117,7 @@ invert_range_selection ranges length needs_sorting =
    Empty subranges are discarded.
 sort_and_merge_ranges : Vector Range -> Vector Range
 sort_and_merge_ranges ranges =
-    sorted = ranges.filter (range-> range.is_empty.not) . sort on=(.start)
+    sorted = ranges.filter (range-> range.is_empty.not) . sort on=(_.start)
```

Comment:
`(.start)` is parsed to `UnresolvedSymbol<start>`, and I found no easy way how to call that. Whereas `(_.start)` is parsed into `Function`. I created issue #6726 to track that.

Reply:
#6276 *